src/cssm.pyx

```diff
@@ -770,15 +770,15 @@ def ddm_flex(np.ndarray[float, ndim = 1] v,
         
         # Drift
         drift_params_tmp = {key: drift_params[key][k] for key in drift_params.keys()}
-        drift[:] = np.add(v_view[k], drift_fun(t = t_s, **drift_params_tmp)).astype(DTYPE)
+        drift_view[:] = np.add(v_view[k], drift_fun(t = t_s, **drift_params_tmp)).astype(DTYPE)
```

Comment:
ok yep this should be better thanks :)

Reply:
(is this potentially wrong across functions in the cssm file? could you apply this across functions?)